packages/send/src/send.ts

```diff
@@ -22,17 +22,19 @@ export const send = <Request extends Req = Req, Response extends Res = Res>(req:
 ): Response => {
   let bodyToSend = body
 
-  // in case of object - turn it to json
-  if (typeof body === 'object' && body !== null) {
+  if (Buffer.isBuffer(body)) {
+    bodyToSend = body
+  } else if (typeof body === 'object' && body !== null) {
+    // in case of object - turn it to json
     bodyToSend = JSON.stringify(body, null, 2)
-  } else {
-    if (typeof body === 'string') {
-      // reflect this in content-type
-      const type = res.getHeader('Content-Type')
+  } else if (typeof body === 'string') {
+    // reflect this in content-type
+    const type = res.getHeader('Content-Type')
 
-      if (type && typeof type === 'string') {
-        res.setHeader('Content-Type', setCharset(type, 'utf-8'))
-      } else res.setHeader('Content-Type', setCharset('text/html', 'utf-8'))
+    if (type && typeof type === 'string') {
```

Comment:
not sure what's the point in adding line breaks?

Reply:
ok, since it's matter of taste I reverted these lines change.

Reply:
to my surprise prettier doesn't add line breaks on multiline blocks yet 🙂